html/pages/ports.inc.php

```diff
@@ -87,9 +87,9 @@
 <?php
 
 if ($_SESSION['userlevel'] >= 5) {
-    $results = dbFetchRows("SELECT `device_id`,`hostname` FROM `devices` GROUP BY `hostname` ORDER BY `hostname`");
```

Comment:
GROUP BY hostname is irrelevant here. hostnames are unique anyway. Same for other queries in this file.